### PR TITLE
Hot Fix for PR #22, unix builds

### PR DESCRIPTION
**Pull request by Miladinho**
Updated unix makefile to create bin directory in the root folder
Updated .gitignore to ignore bin and ignitiond
#22

### CODE REVIEW

**Comment by Miladinho** [comment]
If this works we should update all other supported OS builds

**Comment by Matthematic** [comment]
Can you summarize the issue that this fixed?

**Comment by dragonsmilk** [comment]
Please rebase with dev since https://github.com/ignitioncoin/ignitioncoin/pull/26 was just merged

**Comment by Miladinho** [comment]
@Matthematic 

click the hyper link #22

**Comment by Miladinho** [comment]
@dragonsmilk 

Lets merge things in order next time, I had already added some of the things #26 added to the makefile and now there are some conflicts I have to come through

**Comment by dragonsmilk** [comment]
There would be conflicts either way. The order of PR's in no way dictates dibs on merge order.

**Comment by Miladinho** [comment]
@dragonsmilk 

I understand that, but in this case, if we had these changes in, then he would have been using ignitiond instead of Ignitiond which caused the conflicts, not a big deal, and it was my fault for missing something in the original PR for this. Its just that this PR was sitting here for a while

**Comment by dragonsmilk** [comment]
I wasn't comfortable merging this myself since I didn't quite understand it, while his was a simple .gitignore update for which I was present for in the discord conversation. If we had these changes in, then I would've asked him to rebase instead of you. If nobody else looked at your PR other than me for 4 days, thats not my problem.
